### PR TITLE
Create comparative-politics.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/55343/style-request-comparative-politics

Open article here: http://www.ingentaconnect.com/contentone/cuny/cp/2014/00000047/00000001/art00007

Can you guys help me with the ibid stuff?

### CODE REVIEW

**Comment by rmzelle** [comment]
> Can you guys help me with the ibid stuff?

What specifically do you need help with? The Chicago styles are good examples of how you would usually test for "position" (since cites can have more than one value for "position", you generally test from most specific, i.e. "ibid-with-locator", to least specific, "subsequent"). See e.g. https://github.com/citation-style-language/styles/blob/9e5c5b08e8364d1c6bfc82aac8ceb4399ad85144/chicago-fullnote-bibliography.csl#L1249

**Comment by POBrien333** [comment]
Thanks, @rmzelle 
Each individual citation should have their own number though. Currently it gives me the same number if it was already cited. How can I get it to do that?

**Comment by adam3smith** [comment]
you have this as a numeric in-text style. It should be a "note" style like the Chicago note and full note styles, which will also take care of the numbering issue.

**Comment by POBrien333** [comment]
dang. I was wondering about that. :((

**Comment by POBrien333** [comment]
@adam3smith 
I've just replaced the citation section with the whole bibliography section. That should work, right?

**Comment by rmzelle** [comment]
@adam3smith, metadata looks good to me. Can you merge after reviewing the code?

File: comparative-politics.csl [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      <label form="short" prefix=" (" suffix=")"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name and="text" delimiter-precedes-last="never" initialize-with="."/>
```
Comment (adam3smith) [inline comparative-politics.csl]: I'm seeing Stanley Rosen and Peter Hays Gries, eds. in the instructions, i.e. without `initialize-with`

File: comparative-politics.csl [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+            </group>
+            <text macro="publisher"/>
+          </group>
+        </if>
+        <else-if type="chapter paper-conference" match="any">
+          <text variable="title" quotes="true" suffix=", "/>
+          <text term="in" suffix=" "/>
+          <group delimiter=", ">
+            <text macro="editor"/>
+            <text variable="container-title" form="short" text-case="title" font-style="italic"/>
```
Comment (adam3smith) [inline comparative-politics.csl]: Pretty sure that the `form="short"` shouldn't be here.

File: comparative-politics.csl [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+            <text macro="edition"/>
+          </group>
+          <group delimiter=", " prefix=" ">
```
Comment (adam3smith) [inline comparative-politics.csl]: no need for the prefix -- you have that in the publisher macro already.

File: comparative-politics.csl [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+            </group>
+          </group>
+        </else-if>
+        <else-if type="article article-journal" match="any">
+          <group delimiter=", ">
+            <text variable="title" quotes="true" suffix=","/>
+            <text variable="container-title" text-case="title" font-style="italic"/>
+            <group delimiter=" ">
+              <text variable="volume" font-weight="normal"/>
+              <text macro="year-date" prefix="(" suffix=")"/>
```
Comment (adam3smith) [inline comparative-politics.csl]: instructions suggest this should be month and year? (January 2013)

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**